Update docs with instructions for running tests (#5310)
rotki/rotki: docs/contribute.rst

For running the tests with a more specific usage and invocation, please refer to the `pytest <https://docs.pytest.org/en/stable/usage.html>`__ documentation.

> We run the test suite in the GitHub CI but only a subset of them since not all are needed and sometimes they suffer of rate limiting. We have some special settings to choose what tests are executed:

Reviewer: ```suggestion
We run the test suite in the GitHub CI but only a subset of them since not all are needed and sometimes they suffer from rate limiting. We have some special settings to choose what tests are executed:
```

> - ``[skip py tests]`` will not run the python backend tests regardless of the change status of the monitored directory.

Reviewer: ```suggestion
- ``[skip py tests]`` will not run the python backend tests regardless of whether the backend code has been touched.
```
